packages/augur-ui/src/modules/positions/selectors/positions-summary.ts

```diff
@@ -69,4 +69,4 @@ export const positionSummary = memoize(
   },
 );
 
-const timesHundred = (value) => isNaN(value) ? "0" : createBigNumber(value).times(100);
+const timesHundred = (value) => isNaN(value) ? createBigNumber("0") : createBigNumber(value).times(100);
```

Comment:
@bthaile Is it possible `value` could be a bignumber here?

Reply:
nope it's not because of our Formatting helpers